### ISSUE TITLE
[BUG]:unwanted scrolling and duplicate app drawer

### ISSUE THREAD

**Issue by sambitraze** [Kide-Devs/Kide#35]
## **Describe the bug**
there are two sidebars and also in it there are few parts which are scrolling but the whole side bar shld scrooll instead two part of them are scrolling independently

## **To Reproduce**
1. Go to homescreen
2. Click on open drawer button
3. and Swipe from left on home screen
4. You can see that you are accessing two drawers 

## **Expected behavior**
A static drawer with single child scroll view like behaviour

Duplicate drawer:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/23294656/94968775-28b4e180-051f-11eb-9254-f82592c9f0ce.gif)

Scrolling Issue:

![ezgif com-gif-maker](https://user-images.githubusercontent.com/23294656/94969416-50587980-0520-11eb-8404-b354fb44555b.gif)

**Comment by bibekpanda55** [comment]
@Cryptonex7 I can solve this issue please assign it to me

**Comment by Cryptonex7** [comment]
Sure @bibekpanda55 - We would love to get contributions from you. We will be awaiting your PR.

**Comment by bibekpanda55** [comment]
You have not assigned this issue to me still, I will create a pull request after you assign this issue🙂

**Comment by Cryptonex7** [comment]
well, @bibekpanda55, just for your information, assigning the issue is not mandatory in the opensource flow, yes it is a good practice, we understand that, however, we recommend you to not wait for assignments for your contributions, not just for this repository, but any repository.

At your request, we will assign you the issue, and yes, we will make sure to assign the issues for every request in future, as again, we understand that is is a good practice.

Thanks for bringing this to our notice, we will be awaiting your PR.

**Comment by bibekpanda55** [comment]
Yes got it @Cryptonex7 will keep in mind about this for further PRs and will create a PR for this issue as soon as possible

**Comment by bibekpanda55** [comment]
This error is displayed everytime I try to install the app on my device ,I
tried to solve it but couldn't. Any idea?

**Comment by Cryptonex7** [comment]
@bibekpanda55 sorry,. but can you add a screenshot or the error statement that youre facing?

**Comment by bibekpanda55** [comment]
![Screenshot (53)](https://user-images.githubusercontent.com/53937166/96004325-e366ad00-0e58-11eb-9d4b-738d62de6c17.png)
